webinar.html

```diff
@@ -0,0 +1,231 @@
+---
+layout: page
+title: Webinar 
+permalink: /webinar/
+---
+<style>
+    .taglist
+    {
+        display: none;
+        margin-left: 50px;
+    }
+    input[type=checkbox] {
+        float: left;
+        margin-top: 1.4em;
+        margin-right: 0.5em;
+    }
+    input[type=checkbox]:checked ~ ul {
+        display: block;
+    }
+</style>
+
+<div class="entry-content">
+						<p>Every week, we get to watch 30 to 45 minute webinars on various
+ machine learning topics.  These are the webinars that we
+ watched over the last couple of years.</p>
+ 
+ <ul>
+ 
+<li><strong>2021/03/12-5</strong>: <a href="https://www.youtube.com/watch?v=DhHyshhc1lY">Quantization of Neural Networks – High Accuracy at Low Precision</a> </li>
+<li><strong>2021/02/26</strong>: <a href="https://www.youtube.com/watch?v=jD3L6HiH4ls">Nicholas Carlini: Making and Measuring Progress in Adversarial Machine Learning</a> </li>
+<li><strong>2021/02/12</strong>: <a href="https://www.youtube.com/watch?v=rv5cw2bWx7Q&list=PLuv1FSpHurUcwP9CFLT8efinNGFD7KTVH&index=100"> [NeurIPS 2020] On the Importance of Deconstruction in Machine Learning Research </a> </li>
+<li><strong>2021/02/5</strong>: <a href="https://www.youtube.com/watch?v=t835Cp9hS5k&list=PLuv1FSpHurUcwP9CFLT8efinNGFD7KTVH&index=113&ab_channel=ArtificialIntelligence">[NeurIPS 2020] Leveraging Kinematic for Deep Representation Learning   NewinML </a> </li>
+<li><strong>2021/01/22</strong>: <a href="https://www.youtube.com/watch?v=KWRvE5kFN5I&list=PLuv1FSpHurUcwP9CFLT8efinNGFD7KTVH&index=12&ab_channel=ArtificialIntelligence">[NeurIPS 2020] Layer wise Learning of Kernel Dependence Network Workshop Beyond Backpropagation </a> </li>
+<li><strong>2021/01/15</strong>: <a href="https://www.youtube.com/watch?v=oEgfBILbXEk"> Chundi Liu - Guided Similarity Separation for Image Retrieval  </a> </li>
+
+<li><strong>2020/12/4</strong>: <a href="https://www.youtube.com/channel/UCddp3o-ctW8rmYtfdDfVUkA/videos">   </a>Montreal AI and Neuroscience (MAIN)  </li>
```

Comment:
The title here is not part of the link, so the link is not clickable on the page.
```suggestion
<li><strong>2020/12/4</strong>: <a href="https://www.youtube.com/channel/UCddp3o-ctW8rmYtfdDfVUkA/videos">  Montreal AI and Neuroscience (MAIN) </a>  </li>
```
But this comment is pointless anyway if we decide to switch to embedded videos.